src/clipboard.py

```diff
@@ -1,89 +1,145 @@
-# SPDX-License-Identifier: GPL-3.0-or-later
 import copy
-
 from graphs import graphs, ui
 
 
-class Clipboard:
+class BaseClipboard:
     def __init__(self, application):
```

Comment:
I feel like both undo and redo buttons should be passed along to the base class instead of being defined in each inheriting class